docs/src/installing.rst

```diff
@@ -17,7 +17,8 @@ any WSL_ distributions.
 .. _WSL: https://docs.microsoft.com/en-us/windows/wsl/install-win10
 
 .. note:: Iris currently supports and is tested against **Python 3.6** and
-          **Python 3.7**.
+          **Python 3.7**.  This documentation is built using a single
+          version of Python, version |python_version|.
```

Comment:
@tkknight Could we separate this out into its own `note` ... otherwise the reader can easily skim over it e.g.,
```
.. note:: Iris currently supports and is tested against **Python 3.6** and **Python 3.7**.

.. note:: This documentation was built using Python |python_version|.
```

Reply:
Done.  See 793a219